summit_blood_samples/blood_sample/models.py

```diff
@@ -17,7 +17,8 @@ def __str__(self):
 
     class Meta:
         indexes = [
-            models.Index(fields=['CreatedAt'], name='CreatedAt_bs_import_idx'),
+            models.Index(fields=['CreatedAt'], \
```

Comment:
The backslash isn't needed as you're inside brackets here. In general I'd avoid using a backslash to continue lines (favour bracketing where possible) as it's too easy to make a mistake!

Reply:
Thought of making it bit more readable, will revert it back

Reply:
It's not the multi-line I object to; that's good! But you don't need the backslash at the end of the first line.

Reply:
will revert it